src/test/resources/mapping/expected_marc_record_with_holdings_and_items.mrc

```diff
@@ -0,0 +1 @@
+01611nam a22006013c 4500001000200000005001700002008004100019010001500060020001800075022001500093035001500108035001800123074001500141100002000156245004200176264002900218264003600247300003100283300003600314490003600350490002400386653001600410653003700426653002800463653001300491653005200504655001400556655001300570710002000583711001600603900002100619901000700640902000900647903002200656903002200678904001700700904001700717905002600734906002500760907002100785907002100806908001800827908001800845908001800863908001800881909001300899909001300912909001300925909001300938910000900951910000800960999004100968120200522014642.9190807|20162015||||||||       |||||mul||  a1447294130  a9781447294130  a0262012103  a2003065165  a9780262012102  a05521423521 aFielding, Helen00aThe Journal of ecclesiastical history 1aEdinburghbBirlinnc2015 1aNew YorkbAlfred A. Knopfc2016  axx, 238 p. : ill. ; 24 cm.  axxix, 224 Seiten Illustrationen0 aCooperative information systems0 aThe lateral history  aPoliticians  aPoliticians--Scotland--Biography  aPolitics and government  aScotland  aScotland--Politics and government--21st century 4anewspaper 4atextbook2 aSterling, Roger2 aDraper, Donffa6056.I4588 B749   aPR  a2016  aholdingsLinkText1  aholdingsLinkText212aholdingsUri112aholdingsUri2  aMiller General Stacks  bSpecial Reading Room  a6066.I4588 B749   a6076.I4588 B749   aitemLinkText1  aitemLinkText2  aitemLinkText3  aitemLinkText41 aitemUri11 aitemUri21 aitemUri31 aitemUri4  abook  advdffic8b50e3f-0446-429c-960e-03774b88223f
```

Comment:
Where is this file used? The corresponding file has been changed to a json in the master

Reply:
Removed